st2common/st2common/exceptions/actionalias.py

```diff
@@ -0,0 +1,26 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from st2common.exceptions import StackStormBaseException
+
+__all__ = [
+    'ActionAliasAmbiguityException'
+]
+
+
+class ActionAliasAmbiguityException(StackStormBaseException):
```

Comment:
Should probably inherit from `ValueError`, this way the status code returned by the API on this exception will be correct (400 - bad request).